actors/datacap/src/v12/mod.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019-2022 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use fvm_shared3::bigint::BigInt;
+use fvm_shared3::econ::TokenAmount;
+use fvm_shared3::METHOD_CONSTRUCTOR;
+use lazy_static::lazy_static;
+use num_derive::FromPrimitive;
+
+pub use self::state::State;
+pub use self::types::*;
+
+mod state;
+mod types;
+
+pub const DATACAP_GRANULARITY: u64 = TokenAmount::PRECISION;
+
+lazy_static! {
+    // > 800 EiB
+    pub static ref INFINITE_ALLOWANCE: TokenAmount = TokenAmount::from_atto(
```

Comment:
Do we need this?

Reply:
we are using `INFINITE_ALLOWANCE` and `DATA_CAP_GRANULARITY` in state migration 
@LesnyRumcajs can we use these from here?
https://github.com/ChainSafe/forest/blob/d7efec8fd33747a2cfbd29c01647afbfb7f80fec/src/state_migration/nv17/datacap.rs#L25

Reply:
If those are copied from builtin-actors I'd rather have them here. But it's not a huge deal.

Reply:
it's copied from builtin actors
so i am keeping it